martin/src/config/file/tiles/postgres/builder.rs

```diff
@@ -262,31 +247,20 @@ impl PostgresAutoDiscoveryBuilder {
         let mut used = HashSet::<(&str, &str)>::new();
 
         for (id, cfg_inf) in &self.functions {
-            let Some(db_funcs) = find_info(&db_funcs_info, &cfg_inf.schema, "schema", id) else {
-                continue;
-            };
-            if db_funcs.is_empty() {
-                warn!(
-                    "No functions found in schema {}. Only functions like (z,x,y) -> bytea and similar are considered. See README.md",
-                    cfg_inf.schema
-                );
-                continue;
-            }
-            let func_name = &cfg_inf.function;
-            let Some((pg_sql, db_inf)) = find_info(db_funcs, func_name, "function", id) else {
+            let Ok((merged_inf, pg_sql_info)) =
```

Comment:
nit/unrelated to your change:

One thing that makes this part of the codebase harder to parse is its insistance to hamming code the variable names.

This was previously a quite short name and not very self-explanatory.
Could we lengthen it to something more expressive like `merged_db_and_cfg_info`

Reply:
💯  agreed. I was trying to stick to the existing style, but I'd be _happy to_ improve the naming.

Reply:
I am fine with either, what is simpler for you.

GitHub has a (semi-working) feature where I can go commit by commit when reviewing ^^

Reply:
I've renamed the variables in the two new functions, left the rest for later.